recipes-ni/ni-systemreplication/files/nisystemreplication

```diff
@@ -5,18 +5,28 @@ set -e
 # get or set
 declare -r MODE="${1:-}"
 
+NIRECOVERY_MOUNTPOINT=/mnt/NIRECOVERY
+
 image_get () {
-	nisystemimage getall -x tgz -f getall.tgz
+	mount -o remount,rw $NIRECOVERY_MOUNTPOINT
+	mkdir -p $NIRECOVERY_MOUNTPOINT/replication_images
+	nisystemimage getall -x tgz -f $NIRECOVERY_MOUNTPOINT/replication_images/getall.tgz
```

Comment:
MINOR: Can we figure out a more meaningful name for the archive than `getall.tgz`? Maybe: `latest.tgz`?

Reply:
`latest.tgz` may also cause confusion about what kind of image that is - the one containing just runmode and config (i.e., get operation) or the one containing grub, safemode, runmode, config (i.e., getall).

How about `systemimage.tgz` or `latest_systemimage.tgz`?

Reply:
I'd be fine with either. I suppose the former makes more sense if this workflow isn't supposed to support having multiple images on the tool at once.

Reply:
Done. It's just one image for now but we'll expand it to multiple.